packages/browser-wallet/src/background/recovery.ts

```diff
@@ -45,7 +52,7 @@ async function recoverAccounts(
                     credId,
                     credNumber,
                     status: CreationStatus.Confirmed,
-                    identityIndex,
+                    identityIndex: status.identityIndex || 0,
```

Comment:
I would define:
```
const identityIndex = status.identityIndex || 0;
```
to keep it consistent with what you do for `emptyIndices` and `credNumber` above.